### PR TITLE
Change README format to MarkDown

### PR DESCRIPTION
**Pull request by Jalle19**
This improves the legibility of the README when reading it on Github (which I assume is how most people read it). I've also taken the liberty to change a few words here and there.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,70 +0,0 @@
-=============================
-       Linux, OS-X, BSD
-=============================
-
-Start by executing:
-./bootstrap
-
-To install add-ons into /path/to/XBMC:
```
Comment (opdenkamp) [inline README]: why did you remove this part?

Comment (Jalle19) [inline README]: I thought it was not very clear where exactly the addons will end up being installed. Is the idea to use e.g. `/usr/local/lib/kodi`?

Comment (opdenkamp) [inline README]: correct, so you can add add-ons to an existing xbmc installation

Comment (Jalle19) [inline README]: See the latest commit

**Comment by opdenkamp** [comment]
nice, squash it and merge away

**Comment by Jalle19** [comment]
@opdenkamp squashed

**Comment by MikeB2013** [comment]
Just to confirm, build failure on Linux - Ubuntu, same solution as Millhouse works.
